janusgraph-core/src/main/java/org/janusgraph/graphdb/query/vertex/BasicVertexCentricQueryBuilder.java

```diff
@@ -85,9 +86,12 @@
     private boolean restrict2Partitions = true;
 
 
+    private final BaseVertexCentricQuery queryForLabel;
+
     public BasicVertexCentricQueryBuilder(final StandardJanusGraphTx tx) {
         super(tx);
         this.tx = Preconditions.checkNotNull(tx);
+        this.queryForLabel = queryForLabel();
```

Comment:
Do we really want to construct this query already in the constructor and not simply when it's used for the first time?

Reply:
No, not needed